binaryen-sys/src/lib.rs

```diff
@@ -4,6 +4,8 @@
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 
+pub mod passes;
```

Comment:
I think it sin't a bad idea to separate it, but I'm fine keeping it in the same context (e.g. moving the include here).